src/services/service.services.spec.js

```diff
@@ -0,0 +1,45 @@
+import '../../test/testSetup'
```

Comment:
With this setup, this file would be imported in each spec file. The globals can be not globals and imported per file too